<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dealwindow</class>
 <widget class="QWidget" name="dealwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="closebtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hidebtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="groupch" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>141</width>
     <height>161</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="group1btn">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>9</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="group3btn">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>67</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="group4btn">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>96</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okbtn">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>125</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="group2btn">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>38</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>47</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backbtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="chacklabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>47</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>确认拒绝吗？</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="mask">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>47</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
